python/cucim/src/cucim/skimage/measure/_colocalization.py

```diff
@@ -178,7 +178,7 @@ def manders_coloc_coeff(image0, image1_mask, mask=None):
 
 @cp.fuse()
 def _get_manders_overlap_coeff(image0, image1):
-    denom = (cp.sum(cp.square(image0)) * (cp.sum(cp.square(image1)))) ** 0.5
+    denom = cp.sqrt(cp.sum(cp.square(image0)) * (cp.sum(cp.square(image1))))
```

Comment:
Just trying this out to see how it goes, have already broken it out in a separate PR ( https://github.com/rapidsai/cucim/pull/578 ) if it turns out we want to include it